doc/user_guide/create_new_sale_channel.rst

```diff
@@ -0,0 +1,69 @@
+New Sale Channel
+=================
+
+This module helps you understand how to create a new sale channel.
+
+New Module 
+-----------
+
+On the top, click on the ``Create a new record`` button.
+
+.. figure:: images/23.png
+
+In the window that pops up, you can see a number of fields.
+These are the fields on New Record Window:
+
+* **Name**: The name of your new sales channel. This field is manadatory.
+* **Code**: Optional Field, contains unique code generated during sale. 
+* **Company**: This is mandatory field to fill. It has the name of the company.
+* **Address**: Address where your company is located. This is optional field.
+* **Source**: How your channel will be handled. BY default it is set to Manual
+ entry.
+
+
+.. figure:: images/24.png
+
+Fill all required details and then click on the tab which say ``Order Defaults``.
+This contains your order details. The fields are as follows:
+
+
+* **Warehouse**: Contains details of the warehouse from where your product is
```

Comment:
The warehouse associated with the channel. All new sale orders created from the channel will use this warehouse as the default warehouse to fulfill from. If the channel exports stock levels to an external application (like Amazon or your eCommerce system), the inventory levels exported are of this warehouse.